server/request_sendrawtx.go

```diff
@@ -69,6 +69,9 @@ func (r *RpcRequest) handle_sendRawTransaction() {
 	// Check if transaction needs protection
 	needsProtection := r.doesTxNeedFrontrunningProtection(r.tx)
 
+	// Check if tx is being cached for later bundling
+	cachingTx := r.isCachingBundleTx()
```

Comment:
```suggestion
	_, cachingTx := r.req.URL.Query()["bundle-id"]
```

How about we remove the whole `isCachingBundleTx` function?